sql/core/src/main/scala/org/apache/spark/sql/Dataset.scala

```diff
@@ -1947,6 +1947,21 @@ class Dataset[T] private[sql](
     Except(planWithBarrier, other.planWithBarrier)
   }
 
+  /**
+   * Returns a new Dataset containing rows in this Dataset but not in another Dataset while
+   * preserving the duplicates.
```

Comment:
I think it is good to mention `resolves columns by position (not by name)` here too.